test/Optimizations/runtests.jl

```diff
@@ -29,6 +29,19 @@ using ITensorNetworkAD.ITensorAutoHOOT: batch_tensor_contraction
   end
 end
 
+@testset "test the equivalence of local and line hamiltonian" begin
```

Comment:
We could also add a test here comparing the optimized energy against DMRG or exact diagonalization (maybe there is another test of that).

Reply:
Are you talking about comparing the energy of gradient based methods with DMRG, or just DMRG with different hamiltonians? I guess for the first one the gradient based method performs worse, so not sure whether that could be added as a test

Reply:
I just mean test out that the final energy is correct (how you get the energy is not so important, I just brought up DMRG as an easy method to use to obtain the energy in a different way).

Reply:
It could be a very loose test bound (just check that the first 2 digits of the energy are correct).

Reply:
oh OK, yeah that makes sense

Reply:
done